x-pack/test/apm_api_integration/common/fixtures/es_archiver/apm_8.0.0/mappings.json

```diff
@@ -6213,10 +6213,6 @@
           "read_only_allow_delete": "false"
         },
         "codec": "best_compression",
```

Comment:
Removed `lifecycle` and `refresh_interval` as they crash for serverless & MKI

Reply:
How do we know that removing the `refresh_interval` won’t affect any other tests?

Reply:
as @crespocarlos said, it should pick up the default value and not give any problems, at least, I did that before and nothing happened